test/support/page_objects/visualize_page.js

```diff
@@ -565,12 +565,12 @@ export default class VisualizePage {
     .then(function getRect() {
       return self
       .setFindTimeout(defaultFindTimeout)
-      .findByCssSelector('rect.background')
+      .findByCssSelector('div.chart-wrapper > div.chart > svg')
```

Comment:
I reverted that change.  Not sure what made me make the initial change.  Must have been a problem when running the whole test suite.  But it passed for me now without that change.